src/Authentication/AuthView.vala

```diff
@@ -126,6 +133,14 @@ public class AuthView : Gtk.Widget {
     auth_leaflet.navigate (FORWARD);
   }
 
+  /**
+   * Hides the browser page for when we want to go straight to OOB
+   */
+  public void skip_browser () {
+    browser_page.navigatable = false;
+    auth_leaflet.navigate (FORWARD);
+  }
+
```

Comment:
You mentioned in your [commit message](https://github.com/CodedOre/NewCaw/pull/60/commits/d66d4f651527e1fa02b33e9770365f60fa6ef53a) that on Hachyderm the `cawbird://` redirect does not work. It does work though on `mastodon.online` and `mastodon.social`. So that seems to be a instance-related issue, not a reason to skip that for all of Mastodon.

Reply:
We need a button to bypass it, then. Or I need to install Cawbird (rather than running a dev build from `build/`) and see whether it behaves differently when I have the protocol handler installed. But the console for Twitter definitely prints a warning about not handling the Cawbird URL while Hachyderm prints a warning about not handling its own URL.

Reply:
Ah yes. I remember pressing that now. Maybe I decided to skip that because it never worked for me and I assumed it was a "works on Twitter but not Mastodon" thing.

"Try again with authentication code" or similar wording might be clearer?